### PR TITLE
Integration of HifiBerry media player component.

### PR DESCRIPTION
**Pull request by willholdoway**
## Proposed change

New documentation to integrate a HifiBerry media component in order to allow control of HifiBerry HAT boards on Raspberry Pi's running HifiBerry's own HifiBerry OS.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33741
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1922
- This PR fixes or closes issue: Closes

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/hifiberry.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: HifiBerry
+description: How to set up the HifiBerry media player platform
+logo: hifiberry.png
```
Comment (frenck) [inline source/_integrations/hifiberry.md]: Please remove this line, as it is no longer used.

File: source/_integrations/hifiberry.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ha_category:
+  - Media Player
+ha_release: 0.107.7
+ha_domain: hifiberry
+---
+
+The `HifiBerry` platform allows you to control a [HifiBerry OS](https://www.hifiberry.com/hifiberryos/) media player from Home Assistant. This is an end-to-end streaming lightweight OS built by HifiBerry for their Amp+, DAC+ or Digi+ HAT Raspberry Pi boards compatible with Airplay, Bluetooth, DLNA, LMS/Squeezebox, MPD, Snapcast, Spotify and Roon music services.
+
+To setup, you need specific configuration variables, you can add the following to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_integrations/hifiberry.md]:
```suggestion

## Configuration

To configure, you can add the following to your `configuration.yaml` file:
```

File: source/_integrations/hifiberry.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: hifiberry
+    host: hifiberry.local
+    port: 81
```
Comment (frenck) [inline source/_integrations/hifiberry.md]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/hifiberry.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  default: HifiBerry
+  type: string
+host:
+  description: The IP address or hostname of the device.
+  required: true
+  default: hifiberry.local
+  type: string
+port:
+  description: The Port number of Volumio service.
+  required: true
```
Comment (frenck) [inline source/_integrations/hifiberry.md]: required and having a default?

File: source/_integrations/hifiberry.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+{% configuration %}
+name:
+  description: The name of the device.
+  required: false
+  default: HifiBerry
+  type: string
+host:
+  description: The IP address or hostname of the device.
+  required: true
```
Comment (frenck) [inline source/_integrations/hifiberry.md:32]: According to the code, this option is optional

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, for the PR @willholdoway! 

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/hifiberry.md (hunk 1 above)
Comment (willholdoway) [reply source/_integrations/hifiberry.md]: which line sorry? I can see first four on your comment

Comment (willholdoway) [reply source/_integrations/hifiberry.md]: ah logo, I see

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing the PR as the parent PR has been closed.

**Pull request closed without merging**